example/linux/flutter/CMakeLists.txt

```diff
@@ -82,7 +82,7 @@ add_custom_command(
   COMMAND ${CMAKE_COMMAND} -E env
     ${FLUTTER_TOOL_ENVIRONMENT}
     "${FLUTTER_ROOT}/packages/flutter_tools/bin/tool_backend.sh"
-      linux-x64 ${CMAKE_BUILD_TYPE}
+      ${FLUTTER_TARGET_PLATFORM} ${CMAKE_BUILD_TYPE}
```

Comment:
I'm testing on Flutter 2.0.6 and am not sure whether we should change the `linux-x64`.

Bundled `libusb.so` is x64

Reply:
This is automatically changed by Flutter when you select target platform.

Reply:
Target platform could be Linux-arm which the library doesn't support yet. Linux-x64 is much clear I think